code/drasil-example/Drasil/SSP/GenDefs.hs

```diff
@@ -17,33 +34,6 @@ import Drasil.SSP.Unitals (baseAngle, baseHydroForce, baseLngth, baseWthX,
   shrResI, shrStiffIntsl, shrStress, slcWght, surfAngle, surfHydroForce, 
   surfLoad, totNrmForce, watrForceDif, xi)
 
-import Data.Drasil.Concepts.Documentation (assumption, definition, element, 
-  method_, model, property, system, value, variable)
-import Data.Drasil.Concepts.Math (angle, matrix, normal, perp, surface, vector)
-import Data.Drasil.Concepts.PhysicalProperties (len, mass)
-import Data.Drasil.Concepts.SolidMechanics (normForce, shearForce)
-
-import Data.Drasil.Quantities.Physics (displacement, force)
-import Data.Drasil.Quantities.SolidMechanics (nrmStrss)
-
-import Data.Drasil.SentenceStructures (acroGD, acroT, foldlSent, getTandS, 
-  isThe, ofThe, sAnd)
-
-import Drasil.DocLang.SRS as SRS (physSystLabel)
-
--- Labels
-l1, l2, l3, l4, l5, l6, l7, l8, l9, l10 :: Label
-l1  = mkLabelSame "normForcEq" (Def General)
-l2  = mkLabelSame "bsShrFEq" (Def General)
-l3  = mkLabelSame "resShr" (Def General)
-l4  = mkLabelSame "mobShr" (Def General)
-l5  = mkLabelSame "normShrR" (Def General)
-l6  = mkLabelSame "momentEql" (Def General)
-l7  = mkLabelSame "netForcex" (Def General)
-l8  = mkLabelSame "netForcey" (Def General)
-l9  = mkLabelSame "hookesLaw2d" (Def General)
-l10 = mkLabelSame "displVect" (Def General)
-
 ---------------------------
 --  General Definitions  --
 ---------------------------
```

Comment:
In this file, `GenDefn`s aren't explicitly built and the file exports `RelationConcept`s instead. Should this be changed? @JacquesCarette

Reply:
No, as `RelationConcept` will get revisited, so that can be taken care of at that time.